### PR TITLE
doc: add description for generation of priv_validator_key to config.toml

### PR DESCRIPTION
**Pull request by jaeseung-bae**
## Description

* Add descriptive comment for the config value(`priv_validator_laddr`)

### CODE REVIEW

**Review by 0Tech — APPROVED** [review]

File: config/toml.go [hunk 1]
```diff
@@ -156,6 +156,7 @@ priv_validator_state_file = "{{ js .BaseConfig.PrivValidatorState }}"
 
 # TCP or UNIX socket address for Ostracon to listen on for
 # connections from an external PrivValidator process
+# If this value is set, key file(priv_validator_key.json) will not be generated.
```
Comment (zemyblue) [inline config/toml.go:159]: Is the default value of `priv_validator_laddr` empty string?

Comment (jaeseung-bae) [reply config/toml.go:159]: Yes, default value is empty string.

Comment (jaeseung-bae) [reply config/toml.go:159]: Actually, default value comes from {{ .BaseConfig.PrivValidatorListenAddr }}. which is empty string though.

Comment (zemyblue) [reply config/toml.go:159]: If it is empty string, how about adding the default value explanation also?

**Review by zemyblue — APPROVED** [review]

**Review by tnasu — APPROVED** [review]

**Pull request merged**